TECHMANIA/Assets/Scripts/Serializable/Options.cs

```diff
@@ -246,6 +247,44 @@ public static void RestoreBackup()
     {
         instance = backupInstance;
     }
+
+    public static IEnumerator ResetCustomDataLocation (bool reloadSkin = true)
```

Comment:
This should be an instance method instead of static. Callers can then call `Options.instance.ResetCustomDataLocation`.

Reply:
Please address this comment.